### PR TITLE
[All Hosts] (browsers) Adjust references to IE to account for Edge as…

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
… needed

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[835eca9](https://github.com/OfficeDev/office-js-docs-pr/commits/835eca9196496acfb08bb470125b30de2da94e27)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/images/dk2-agave-overview-02.png](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/images/dk2-agave-overview-02.png) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909070011249441-1295%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jJEYWB5i9FaSvHASMc5CkvSiubLtP0MsyXjGMq6ZqPA%3D&st=2019-09-07T00%3A07%3A31Z&se=2019-10-08T00%3A12%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/attach-debugger-from-task-pane.md [hunk 1]
```diff
@@ -27,13 +27,14 @@ In Visual Studio, you will see the code files in **Solution Explorer**.   You ca
 > If you don't see the Personality menu, you can debug your add-in using Visual Studio. Ensure your task pane add-in is open in Office, and then follow these steps:
 >
 > 1. In Visual Studio, choose **DEBUG** > **Attach to Process**.
-> 2. In **Attach to Process**, choose all of the available Iexplore.exe processes, and then choose the **Attach** button.
+> 2. In the **Available processes**, choose *either* all of the available `Iexplore.exe` processes *or* all of the available `MicrosoftEdge*.exe` process, depending on [whether your add-in is using IE or Edge](../concepts/browsers-used-by-office-web-add-ins.md), and then choose the **Attach** button.
```
Comment (kbrandl) [inline docs/testing/attach-debugger-from-task-pane.md]:
```suggestion
> 2. In **Available processes**, choose *either* all of the available `Iexplore.exe` processes *or* all of the available `MicrosoftEdge*.exe` processes, depending on [whether your add-in is using Internet Explorer or Microsoft Edge](../concepts/browsers-used-by-office-web-add-ins.md), and then choose the **Attach** button.
```

File: docs/concepts/privacy-and-security.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ The following sections briefly describe how the runtime architecture supports ru
...
+In supported clients for desktop and tablet devices, such as Excel on Windows, and Outlook on Windows and Mac, Office Add-ins are supported by integrating an in-process component, the Office Add-ins runtime, which manages the add-in lifecycle and enables interoperability between the add-in and the client application. The add-in webpage itself is hosted out-of-process. As shown in figure 1, on a Windows desktop or tablet device, [the add-in webpage is hosted inside an Internet Explorer or Edge control](browsers-used-by-office-web-add-ins.md) which, in turn, is hosted inside an add-in runtime process that provides security and performance isolation.
```
Comment (kbrandl) [inline docs/concepts/privacy-and-security.md]:
```suggestion
In supported clients for desktop and tablet devices, such as Excel on Windows, and Outlook on Windows and Mac, Office Add-ins are supported by integrating an in-process component, the Office Add-ins runtime, which manages the add-in lifecycle and enables interoperability between the add-in and the client application. The add-in webpage itself is hosted out-of-process. As shown in figure 1, on a Windows desktop or tablet device, [the add-in webpage is hosted inside an Internet Explorer or Microsoft Edge control](browsers-used-by-office-web-add-ins.md) which, in turn, is hosted inside an add-in runtime process that provides security and performance isolation.
```

File: docs/project/project-add-ins.md [hunk 3]
```diff
@@ -140,7 +140,7 @@ In Project 2013, you can install add-ins as stand-alone solutions on a file shar
 There can be multiple add-in manifest XML files and subdirectories in a file share. You can add or remove manifest directory locations and catalogs by using the  **Trusted Add-in Catalogs** tab in the **Trust Center** dialog box in Project 2013. To show an add-in in Project, the **SourceLocation** element in a manifest must point to an existing website or HTML source file.
 
 > [!NOTE]
-> Internet Explorer 9 (or later) must be installed, but does not have to be the default browser. Office Add-ins require components in Internet Explorer 9. The default browser can be Internet Explorer 9, Safari 5.0.6, Firefox 5, Chrome 13, or a later version of one of these browsers.
+> If you are developing on a Windows computer, either Internet Explorer or Edge must be installed. For more information see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```
Comment (kbrandl) [inline docs/project/project-add-ins.md]:
```suggestion
> If you are developing on a Windows computer, either Internet Explorer or Microsoft Edge must be installed. For more information see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

File: docs/testing/attach-debugger-from-task-pane.md [hunk 4]
```diff
@@ -27,13 +27,14 @@ In Visual Studio, you will see the code files in **Solution Explorer**.   You ca
...
 
 For more information about debugging in Visual Studio, see the following:
 
 -	To launch and use the DOM Explorer in Visual Studio, see Tip 4 in the [Tips and Tricks](https://blogs.msdn.microsoft.com/officeapps/2013/04/16/building-great-looking-apps-for-office-using-the-new-project-templates/#tips_tricks) section of the [Building great-looking apps for Office using the new project templates](https://blogs.msdn.microsoft.com/officeapps/2013/04/16/building-great-looking-apps-for-office-using-the-new-project-templates) blog post.
 -	To set breakpoints, see [Using Breakpoints](/visualstudio/debugger/using-breakpoints?view=vs-2015).
 -	To use F12, see [Using the F12 developer tools](/previous-versions/windows/internet-explorer/ie-developer/samples/bg182326(v=vs.85)).
+-   To use the Edge Developer Tools, see [Microsoft Edge DevTools](https://www.microsoft.com/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?activetab=pivot%3Aoverviewtab).
```
Comment (kbrandl) [inline docs/testing/attach-debugger-from-task-pane.md]:
```suggestion
-   To use the Microsoft Edge Developer Tools, see [Microsoft Edge DevTools](https://www.microsoft.com/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?activetab=pivot%3Aoverviewtab).
```

File: docs/testing/testing-and-troubleshooting.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ You can also give us feedback: if using Excel on Windows or Mac, you can report
 
 ## Outlook add-in doesn't work correctly
 
-If an Outlook add-in running on Windows is not working correctly, try turning on script debugging in Internet Explorer. 
+If an Outlook add-in running on Windows, and [using Internet Explorer](../concepts/browsers-used-by-office-web-add-ins.md), is not working correctly, try turning on script debugging in Internet Explorer.
```
Comment (kbrandl) [inline docs/testing/testing-and-troubleshooting.md]:
```suggestion
If an Outlook add-in running on Windows and [using Internet Explorer](../concepts/browsers-used-by-office-web-add-ins.md) is not working correctly, try turning on script debugging in Internet Explorer. 
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[220b04a](https://github.com/OfficeDev/office-js-docs-pr/commits/220b04a21a13674785da379586286487c10ccd39)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/images/dk2-agave-overview-02.png](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/images/dk2-agave-overview-02.png) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909092046400730-1295%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9JGValnXm7pfRLmChCICMpC68wZoadTDPSnknWItqHQ%3D&st=2019-09-09T20%3A42%3A46Z&se=2019-10-10T20%3A47%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[787b8e1](https://github.com/OfficeDev/office-js-docs-pr/commits/787b8e170823a873f68f07c765c0babef5beeafa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/images/dk2-agave-overview-02.png](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/images/dk2-agave-overview-02.png) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909092047509576-1295%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=nyuY7OHmiU0C894xQJlq4H1X2H7GvBmlXgZZ5Q2dhM8%3D&st=2019-09-09T20%3A43%3A54Z&se=2019-10-10T20%3A48%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b3e06d8](https://github.com/OfficeDev/office-js-docs-pr/commits/b3e06d889ac3741a3b873277cfd49f7c12a7bb17)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/images/dk2-agave-overview-02.png](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/images/dk2-agave-overview-02.png) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909092048108802-1295%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2IDlRvSJXlcZxI7o7aEFlNNyobjz6In1BvtdDLRIFPc%3D&st=2019-09-09T20%3A45%3A05Z&se=2019-10-10T20%3A50%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8deafd3](https://github.com/OfficeDev/office-js-docs-pr/commits/8deafd38343f706f5d05ee12e9434fe72763ef11)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/images/dk2-agave-overview-02.png](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/images/dk2-agave-overview-02.png) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909092049171455-1295%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2Fd5nqfvA9CLiAqeuM0pyq40OrtjXHs7mVi70T2Sx6JM%3D&st=2019-09-09T20%3A46%3A15Z&se=2019-10-10T20%3A51%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**